packages/ember-glimmer/lib/syntax/input.ts

```diff
@@ -1,12 +1,15 @@
 /**
 @module ember
 */
+import * as WireFormat from '@glimmer/wire-format';
 import { assert } from 'ember-debug';
 import { wrapComponentClassAttribute } from '../utils/bindings';
 import { dynamicComponentMacro } from './dynamic-component';
 import { hashToArgs } from './utils';
+import { OpcodeBuilderDSL } from '@glimmer/runtime';
+import { unwrap } from '@glimmer/util';
```

Comment:
I believe this is stripped in glimmer-vm, but I don’t believe that is true here.

Reply:
I had that question too, @chadhietala probably knows that stuff the most intimately.